.github/workflows/zoom-upload.yml

```diff
@@ -0,0 +1,46 @@
+name: Upload Zoom Recordings to YouTube
+
+on:
+  schedule:
+    # 10 minutes after every hour before noon on weekdays
+    # TODO: when we are ready to switch off Circle, run every hour instead of
+    # just in the morning.
+    - cron: "10 0-12 * * 1-5"
+
+  workflow_dispatch: {}
+
+  # XXX: This is for testing; do not merge
+  pull_request: {}
+
+jobs:
+  zoom_to_youtube:
+    name: Upload Zoom to YouTube
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+          cache: 'pip'
+
+      - name: Install Dependencies
+        run: pip install -r requirements.txt
```

Comment:
I didn’t bother with using a virtual environment here like we do on Circle, but you could if you want.